website/source/docs/guides/minikube.md

```diff
@@ -0,0 +1,259 @@
+---
+
+# Consul Installation to Minikube via Helm
+
+<script src="https://fast.wistia.com/embed/medias/qwhi1gvkeq.jsonp" async></script><script src="https://fast.wistia.com/assets/external/E-v1.js" async></script><div class="wistia_responsive_padding" style="padding:56.25% 0 0 0;position:relative;"><div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;"><div class="wistia_embed wistia_async_qwhi1gvkeq videoFoam=true" style="height:100%;position:relative;width:100%"><div class="wistia_swatch" style="height:100%;left:0;opacity:0;overflow:hidden;position:absolute;top:0;transition:opacity 200ms;width:100%;"><img src="https://fast.wistia.com/embed/medias/qwhi1gvkeq/swatch" style="filter:blur(5px);height:100%;object-fit:contain;width:100%;" alt="" onload="this.parentNode.style.opacity=1;" /></div></div></div></div>
+
+In this guide, you'll start a local Kubernetes cluster with minikube. You'll install Consul with only a few commands, then deploy two custom services that use Consul to discover each other over encrypted TLS via Consul Connect. Finally, you'll tighten down Consul Connect so that only the approved applications can communicate with each other.
+
+[Demo code](https://github.com/hashicorp/demo-consul-101) is available.
+
+- [Task 1: Start Minikube and Install Consul with Helm](#task-1-start-minikube-and-install-consul-with-helm)
+- [Task 2: Deploy a Consul Aware Application to the Cluster](#task-2-deploy-a-consul-aware-application-to-the-cluster)
+- [Task 3: Configure Consul Connect](#task-3-use-consul-connect)
+
+## Prerequisites
+
+Let's install Consul on Kubernetes with minikube. This is a relatively quick and easy way to try out Consul on your local machine without the need for any cloud credentials. You'll be able to use most Consul features right away.
+
+First, you'll need to follow the directions for [installing minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/), including VirtualBox or similar.
+
+You'll also need to install `kubectl` and `helm`.
+
+Mac users can install `helm` and `kubectl` with Homebrew.
+
+```sh
+$ brew install kubernetes-cli
+$ brew install kubernetes-helm
+```
+
+Windows users can use Chocolatey with the same package names:
+
+```sh
+$ choco install kubernetes-cli
+$ choco install kubernetes-helm
+```
+
+For more on Helm, see [helm.sh](https://helm.sh/).
+
+## Task 1: Start Minikube and Install Consul with Helm
+
+### Step 1: Start Minikube
+
+Start minikube. You can use the `--memory` option with the equivalent of 4GB to 8GB so there is plenty of memory for all the pods we will run. This may take several minutes. It will download a 100-300MB of dependencies and container images.
+
+```
+$ minikube start --memory 4096
+```
+
+Next, let's view the local Kubernetes dashboard with `minikube dashboard`. Even if the previous step completed successfully, you may have to wait a minute or two for minikube to be available. If you see this message, try again.
```

Comment:
"if you see this message, try again"

Which message? Should there be a specific error message here?

Reply:
Ah, good catch. That was a transcription from the video and wasn't edited out. Will fix.

Reply:
Fixed in a2d0b7c41